eclipse-scout-core/src/table/columns/NumberColumnModel.ts

```diff
@@ -13,6 +13,12 @@ export interface NumberColumnModel extends ColumnModel<number> {
   aggregationFunction?: NumberColumnAggregationFunction;
   backgroundEffect?: 'colorGradient1' | 'colorGradient2' | 'barChart';
   decimalFormat?: DecimalFormat | string | DecimalFormatOptions;
+  /**
+   * Passed to the editor.
```

Comment:
Please write more explicit, that it has no effect on the column itself but only passed to the cell editor field, if the cell is editable.